src/status_im/subs/communities.cljs

```diff
@@ -368,8 +374,8 @@
                                                 checking?
                                                 token-images)
                                        tokenRequirement))
-                                    (or (:criteria highest-role)
-                                        (:criteria lowest-role)))})))
+                                    (visible-criteria (or (:criteria highest-role)
+                                                          (:criteria lowest-role))))})))
```

Comment:
The or clause will return a single value and not a sequence, Therefore `visible-criteria` will only evaluate a single item and not a sequence and it might be a `boolean`.

I am wondering how did that work.
```clj
user=> (every? true? (or true false))
Execution error (IllegalArgumentException) at user/eval164 (REPL:1).
Don't know how to create ISeq from: java.lang.Boolean
```
I ran it in a Java repl.